code/src/types/common.ts

```diff
@@ -42,3 +42,7 @@ export type PaginationParams = {
 export type RTKQueryMockState<TData = null, TError = null> = Partial<
   RTKQueryReturnState<TData, TError>
 >;
+
+export type UserIdType = {
```

Comment:
> Add this type to our UserTypes. Name it UserId and use it for existing types ![image](https://private-user-images.githubusercontent.com/118522791/347907979-4a93d101-591a-4324-b507-9721bcf5ff30.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA3MjE1MTAsIm5iZiI6MTcyMDcyMTIxMCwicGF0aCI6Ii8xMTg1MjI3OTEvMzQ3OTA3OTc5LTRhOTNkMTAxLTU5MWEtNDMyNC1iNTA3LTk3MjFiY2Y1ZmYzMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzExJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcxMVQxODA2NTBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04OTYzZjA1OGY1MDVlZjU3YWNkNTE4ZjljODQ4ZTE1MzQzZjU3OTg1N2JlNjNlNWY0ZDgxOTJkMWZkZDRjODQ1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.3L_ptQFIMhRR8STHz2q0kTss0zQt3Vlv2yWCmBkaT0k)

I decided to pass this type to `common` because this type uses for cart items and also orders the same type
So it's `common`

Reply:
UserId related exactly to userType. If you are using it with other entities it does not mean that this type is common. A common type is for general things. Also, look at the name of this type.  The name of this type starts with `User` . Also, you can ask the team if you do not trust me )

Reply:
> `UserId` is related exactly to the `User` entity. If you are using it with other entities it does not mean that this type is common. A common type is for general things. Look at the name of this type, the name of this type starts with `User` . Also, you can ask the team if you do not trust me ) According to the naming, we are not using ending `Type` for union types

It's `common` type and reusable. Is not it ? 
Otherwise we can ask our mentors and experts